### PR TITLE
fixed the Doctype whitespace and added body tag

### PR DESCRIPTION
**Pull request by Blessings9**

### CODE REVIEW

**Comment by Blessings9** [comment]
rebase from Master and update the pull request

**Comment by Paxilio** [comment]
please rebase of Master and update pull request

**Comment by Blessings9** [comment]
i have rebased, review changes please

File: index.html [hunk 1]
```diff
@@ -1,10 +1,12 @@
- < !DOCTYPE html>
```
Comment (Paxilio) [inline index.html:1]: totally missed that...thanks mate

**Pull request merged**